manager/integration/tests/common.py

```diff
@@ -2499,31 +2523,30 @@ def is_backupTarget_nfs(s):
 
 
 def find_backup(client, vol_name, snap_name):
-    found = False
-    for i in range(100):
+    """
+    find_backup will look for a backup on the backupstore
+    it's important to note, that this can only be used for completed backups
+    since the backup.cfg will only be written once a backup operation has
+    been completed successfully
+    """
+
+    def find_backup_volume():
         bvs = client.list_backupVolume()
         for bv in bvs:
             if bv.name == vol_name:
-                found = True
-                break
-        if found:
-            break
-        time.sleep(1)
-    assert found
-
-    found = False
-    for i in range(20):
-        backups = bv.backupList().data
-        for b in backups:
-            if b.snapshotName == snap_name:
-                found = True
-                break
-        if found:
-            break
-        time.sleep(1)
-    assert found
-
-    return bv, b
+                return bv
+        return None
+
+    for i in range(120):
+        bv = find_backup_volume()
```

Comment:
Can we set ```bv``` to ```None``` initially and find backup only if ```bv``` is ```None```? That way we can avoid calling the method ```find_backup_volume``` after we have the value of ```bv``` already.

Reply:
We can but initially the volume.cfg will not be present, since the BV existence is based on the existence of a folder.
This shouldn't have an impact since calling backupList() on a non existent backupVolume should just return an empty list.

Reply:
Will change it.